docs/_posts/api/v3.3.1/0200-01-01-l-mapbox-map.html

```diff
@@ -25,7 +25,7 @@ <h2 id="section-l-mapbox-map">L.mapbox.map(element, id|url|tilejson, options)</h
 <tr>
 <td>id <em>or</em> url <em>or</em> tilejson</td>
 <td><strong>string</strong> if <em>id</em> or <em>url</em> <strong>object</strong> if <em>tilejson</em></td>
-<td>url can be <ul><li>A map <code>id</code> string <code>examples.map-foo</code></li><li>A comma separated list of map <code>id</code> strings <code>examples.map-foo,examples.map-bar</code> <a href="https://www.mapbox.com/mapbox.js/example/v1.0.0/compositing/">example</a></li><li>A URL to TileJSON, like <code>{{site.tileApi}}/v3/mapbox.dark.json</code></li><li>A <a href="https://docs.mapbox.com/api/maps/#retrieve-tilejson-metadata">TileJSON</a> object, from your own Javascript code</li></ul></td>
+<td>url can be <ul><li>A map <code>id</code> string <code>examples.map-foo</code></li><li>A comma separated list of map <code>id</code> strings <code>examples.map-foo,examples.map-bar</code> <a href="https://www.mapbox.com/mapbox.js/example/v1.0.0/compositing/">example</a></li><li>A URL to TileJSON, like <code>{{site.tileApi}}/v3/mapbox.satellite.json</code></li><li>A <a href="https://docs.mapbox.com/api/maps/#retrieve-tilejson-metadata">TileJSON</a> object, from your own Javascript code</li></ul></td>
 </tr>
 <tr>
 <td>options</td>
```

Comment:
Line 33 contains two links to L.mapbox.gridLayer, should those be removed, too?

Reply:
👍 great catch 8dd17dde